enterprise/api/v1/views.py

```diff
@@ -27,6 +28,20 @@ class EnterpriseReadOnlyModelViewSet(viewsets.ReadOnlyModelViewSet):
     throttle_classes = (ServiceUserThrottle,)
 
 
+class EnterpriseReadOnlyModelViewSet(EnterpriseModelViewSet, viewsets.ReadOnlyModelViewSet):
+    """
+    Base class for all read only Enterprise model view sets.
+    """
+    pass
+
+
+class EnterpriseReadWriteModelViewSet(EnterpriseModelViewSet, viewsets.ModelViewSet):
```

Comment:
@bdero No worries, I closed #36. It looks like this PR contains everything required and is far more complete than #36.